kiwi/partitioner/gpt.py

```diff
@@ -79,3 +79,8 @@ def set_flag(self, partition_id, flag_name):
             )
         else:
             log.warning('Flag %s ignored on GPT', flag_name)
+
+    def set_hybrid_mbr(self):
+        Command.run(
+            ['sgdisk', '-h', '1:2:3', self.disk_device]
```

Comment:
sure boot code stays the same as it is right now, if this works for you at the moment it will work afterwards too. actually we just move one part of your custom script into kiwi not more